src/include/stir/recon_buildblock/PoissonLogLikelihoodWithLinearKineticModelAndDynamicProjectionData.txx

```diff
@@ -304,7 +304,7 @@ set_up_before_sensitivity(shared_ptr<TargetT > const& target_sptr)
       return Succeeded::no;
     }
 
-  shared_ptr<ProjDataInfo> proj_data_info_sptr(
+  const shared_ptr<ProjDataInfo> proj_data_info_sptr(
```

Comment:
not really needed, and maybe somewhat confusing. (it's a const pointer to a non-const object), but as you prefer